source/parser/parser.cpp

```diff
@@ -179,7 +181,7 @@ void Parser::Run()
 {
     int         error_line = -1;
     int         error_col = -1;
-    UCS2String  error_filename(MAX_PATH, 0); // Pre-claim some memory, so we can handle an out-of-memory error.
+    UCS2String  error_filename(POV_FILENAME_BUFFER_CHARS, 0); // Pre-claim some memory, so we can handle an out-of-memory error.
```

Comment:
Technically this change reduces the size of the buffer by 1. However, the `UCS2String` constructor expects the parameter to exclude the terminating NUL character, so the new code should more accurately reflect the original intention.